src/ws_server.cu

```diff
@@ -44,6 +43,13 @@ DEFINE_string(cal_file, "", "path name to calibration file");
 
 enum ExposureMode { AUTO = 0, MANUAL = 1 };
 
+class CameraException : public std::exception {
```

Comment:
Do you think it would be useful to store the exception in this file instead of in a different file?

Reply:
good suggestion, thanks.  Done.